projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_grid.scss

```diff
@@ -596,3 +596,217 @@ $_bootstrap-grid: extend(
+            igx-color: 'warn',
+        ),
+
+        filtering-background-or--focus: (
+            igx-color: 'warn',
+            darken: 8%
+        ),
+
+        edited-row-indicator: (
+            igx-color: ('primary', 500)
+        ),
+
+        cell-selected-background: (
+            igx-color: ('primary', 50),
+            rgba: .5
+        ),
+
+        cell-active-border-color: (
+            igx-color: ('primary', 500)
+        ),
+
+        ghost-header-background: (
+            igx-color: ('primary', 500)
+        ),
+
+        ghost-header-text-color: (
+            igx-contrast-color: ('primary', 500)
+        ),
+
+        ghost-header-icon-color: (
+            igx-contrast-color: ('primary', 500)
+        ),
+
+        row-border-color: (
+            igx-color: ('primary', 50)
+        ),
+
+        row-highlight: (
+            igx-color: ('primary', 500)
+        ),
+
+        row-ghost-background: (
+            igx-color: ('primary', 50)
+        ),
+
+        row-hover-background: (
+            igx-color: ('primary', 50),
+            rgba: .12,
+            to-opaque: (
+                igx-color: 'surface'
+            )
+        ),
+
+        sorted-header-icon-color: (
+            igx-color: ('primary', 500)
+        ),
+
+        sortable-header-icon-hover-color: (
+            igx-color: ('primary', 500)
+        ),
+
+        row-selected-background: (
+            igx-color: ('primary', 50),
+            rgba: .72,
+            to-opaque: ()
+        ),
+
+        row-selected-hover-background: (
+            igx-color: ('primary', 50),
+        ),
+
+        pinned-border-color: (
+            igx-color: ('primary', 500)
+        ),
+
+        pinned-border-width: rem(4px),
+
+        resize-line-color: (
+            igx-color: ('primary', 500)
+        ),
+
+        drop-indicator-color: (
+            igx-color: ('primary', 500)
+        ),
+
+        edit-mode-color: (
+            igx-color: ('primary', 500)
+        ),
+
+        grouparea-background: (
+            igx-color: ('grays', 100)
+        ),
+
+        grouparea-color: (
+            igx-color: ('primary', 500)
+        ),
+
+        drop-area-background: (
+            igx-color: ('grays', 200)
+        ),
+
+        drop-area-text-color: (
+            igx-contrast-color: ('grays', 200),
+            rgba: .62
+        ),
+
+        drop-area-icon-color: (
+            igx-contrast-color: ('grays', 200),
+            rgba: .62
+        ),
+
+        root-summaries-background: (
+            igx-color: ('grays', 100),
+            to-opaque: ()
+        ),
+
+        body-summaries-background: (
+            igx-color: ('grays', 100),
+            to-opaque: ()
+        ),
+
+        group-row-background: (
+            igx-color: ('grays', 100),
+            to-opaque: ()
+        ),
+
+        group-row-selected-background: (
+            igx-color: ('grays', 100),
+            to-opaque: ()
+        )
+    )
+);
+
```

Comment:
According to the handoff, the 'group-count-background' color should be 'primary', the 'group-count-text-color' - white, and the 'expand-icon-color' - 'primary', 200